src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule_cars.java

```diff
@@ -2,8 +2,8 @@
 
 import java.net.URI;
 
-public abstract  class GoGoDuckModule_cars extends GoGoDuckModule {
-    private static final String CARS_FILENAME = "CARS2009_Australia_weekly.nc";
+public abstract class GoGoDuckModule_cars extends GoGoDuckModule {
+    private static final String CARS_FILENAME = "AODN/Australian_Government/CSIRO/Climatology/CARS/2009/eMII-product/CARS2009_Australia_weekly.nc";
```

Comment:
Or does the location of this dataset on http://data.aodn.org.au/ need to change?